fMRISurface/scripts/RibbonVolumeToSurfaceMapping.sh

```diff
@@ -54,7 +54,7 @@ log_Msg "START"
 
 WorkingDirectory="$1"
 VolumefMRI="$2"
-Subject="$3"
+Session="$3"
```

Comment:
When the internal logic of a script doesn't need to change, I'm not sure we need to go to the trouble of replacing the variable names.  I'm not saying undo this, but maybe don't do it for future scripts that don't have functionality changes.

Reply:
Agree, though perhaps not worth changing it back...